### ISSUE TITLE
[Bug]: In Subscription Subtitle Text Not Visible in Light Mode

### ISSUE THREAD

**Issue by Sindhuu-B** [sayeeg-11/OpenSource-Compass#785]
### Bug Description

The subtitle text below “Stay Ahead of Open Source” is barely visible in light mode due to insufficient color contrast between the text and the background.

### Steps to Reproduce

1. Switch to Light Mode
2. Scroll below till Subscribe modal
3. See that the text written below Stay Ahead of Open Source not visible

### Expected Behavior

The subtitle text should be clearly readable in light mode with proper contrast that meets accessibility standards.

### Actual Behavior

The subtitle appears very faint and blends into the background, making it difficult to read.

### Browser

Chrome

### Operating System

Windows 11

### Logs / Screenshots

In Dark Mode
<img width="1772" height="772" alt="Image" src="https://github.com/user-attachments/assets/6760298d-b5ee-4a92-9c88-618b858ce7f3" />

In light Mode
<img width="1798" height="811" alt="Image" src="https://github.com/user-attachments/assets/3432768d-0194-4db7-8ad1-c4b7bec93443" />

I am SWOC'26 Contributor and would love to take this issue. @sayeeg-11
